mmv1/products/containerattached/terraform.yaml

```
@@ -0,0 +1,64 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Cluster: !ruby/object:Overrides::Terraform::ResourceOverride
+    # Don't generate the LRO handling code because it doesn't account for regional services.
+    autogen_async: false
+    id_format: "projects/{{project}}/locations/{{location}}/attachedClusters/{{name}}"
+    import_format: ["projects/{{project}}/locations/{{location}}/attachedClusters/{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "container_attached_cluster_minimal"
```

by convention we call this the "basic" test

```suggestion
        name: "container_attached_cluster_basic"
```

---

Updated.